includes/class-wc-shipping.php

```diff
@@ -160,8 +160,8 @@ public function load_shipping_methods( $package = array() ) {
 			$this->shipping_methods = $shipping_zone->get_shipping_methods( true );
 
 			// Debug output.
-			if ( $debug_mode && ! defined( 'WOOCOMMERCE_CHECKOUT' ) && ! defined( 'WC_DOING_AJAX' ) && ! wc_has_notice( 'Customer matched zone "' . $shipping_zone->get_zone_name() . '"' ) ) {
-				wc_add_notice( 'Customer matched zone "' . $shipping_zone->get_zone_name() . '"' );
+			if ( $debug_mode && ! defined( 'WOOCOMMERCE_CHECKOUT' ) && ! defined( 'WC_DOING_AJAX' ) && ! wc_has_notice( sprintf( esc_html__( 'Customer matched zone "%s"', 'woocommerce' ), $shipping_zone->get_zone_name() ) ) ) {
+				wc_add_notice( sprintf( esc_html__( 'Customer matched zone "%s"', 'woocommerce' ), $shipping_zone->get_zone_name() ) );
```

Comment:
I don't think it is necessary to escape the string here. The string will be escaped before output in https://github.com/woocommerce/woocommerce/blob/5b0cc306a588d4a6549862f6dcc5a1151002294d/includes/wc-notice-functions.php#L147

Also, could you please fix the PHPCS error? https://travis-ci.org/woocommerce/woocommerce/jobs/569660311#L505

Since we are working on this, could you please create a variable to hold the message and use it when calling `wc_has_notice()` and `wc_add_notice()`? This way we avoid declaring the same string twice and the code should become a bit easier to read.

Reply:
Removed escaping and fixed all the errors.